packages/Breadcrumbs/stories/examples/AllVariations.js

```diff
@@ -83,6 +83,42 @@ const AllVariations = () => {
       <Breadcrumbs>
         <Breadcrumbs.Link href={URL}>Breadcrumb 1</Breadcrumbs.Link>
       </Breadcrumbs>
+
+      <Gap />
+
+      <Heading level={3} displayLevel={5} isLight>
+        Not clickable breadcrumb item
+      </Heading>
+
+      <Breadcrumbs>
+        <Breadcrumbs.Link
+          as="span"
+          style={{
+            fontWeight: "bold",
+            color: "#3f3d3c",
+          }}
+        >
+          Breadcrumb 1 with long content, Breadcrumb 1 with long content.
+        </Breadcrumbs.Link>
+        <Breadcrumbs.Link href={URL}>Breadcrumb 2</Breadcrumbs.Link>
+        <Breadcrumbs.Link href={URL}>Breadcrumb 3</Breadcrumbs.Link>
+        <Breadcrumbs.Link href={URL}>Breadcrumb 4 with long content, Breadcrumb 4 with long content.</Breadcrumbs.Link>
+      </Breadcrumbs>
+
+      <Gap />
+
+      <Heading level={3} displayLevel={5} isLight>
+        Customized breadcrumb link
+      </Heading>
+
+      <Breadcrumbs>
+        <Breadcrumbs.Link as="a" href="www.google.com">
```

Comment:
Small thing but the URL needs `http://` or it appends to the current URL.

Again, I think the text should just be Breadcrumb 1 and Breadcrumb 4 (not long text) to keep the stories simple.

Reply:
Sure